drivers/unix/dir_access_unix.cpp

```diff
@@ -227,17 +229,17 @@ static void _get_drives(List<String> *list) {
 			list->push_back(home);
 		}
 
-		// Check $HOME/.config/gtk-3.0/bookmarks
 		char path[1024];
 		snprintf(path, 1024, "%s/.config/gtk-3.0/bookmarks", home);
 		FILE *fd = fopen(path, "r");
+		String fpath;
```

Comment:
Why declare `fpath` outside of the while loop? There isn't really a purpose to have it exist in this scope, the value isn't re-used across loops, and the additional usage that this PR adds is completely unrelated.

You should change this back to the way it was, and for the new behavior, change `fpath =` to `String desktop_path =` etc.

Reply:
Fixed.